cmd/containerd/command/service_windows.go

```diff
@@ -379,3 +384,52 @@ func removePanicFile() {
 		}
 	}
 }
+
+func chunkedFileLog(path string) error {
```

Comment:
I think we can modify this function here: https://github.com/kevpar/containerd/blob/fork/release/1.4/pkg/ioutil/ioutil.go#L27, to take `offset` and the `numberOfBytes` to read and then just read that much info from that file. Then we can use that function here. What do you think?

Reply:
wouldn't this mean that we'd need to open file every single time to read next chunk?

Reply:
1. Seems like we need to rectify different approaches both in how we chunk the file and if we log one stack entry like Kevin's change vs log chunks.

2. I agree it makes sense to only open the file once and read in chunks from the same file object vs reopening the file for every read.

3. It probably makes sense to buffer larger pieces of the file. It is currently reading 4k at a time.